### PR TITLE
Set up KAFKA environment variables for the service execution

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

File: roles/amq_streams_broker/tasks/service.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@
     service_systemd_env_file: "{{ amq_streams_broker_service_config_file }}"
     server_java_opts: "{{ amq_streams_broker_java_opts | default('') }}"
     server_java_heap_opts: "{{ amq_streams_broker_java_heap_opts | default('') }}"
+    server_java_performance_opts: "{{ amq_streams_broker_java_performance_opts | default('') }}"
```
Comment (rmarting) [inline roles/amq_streams_broker/tasks/service.yml:29]: using `default('')` will not create the variable with an empty entry? I mean, the idea is only to declare the Kafka variables only if there is a value to add. Maybe I am wrong about Ansible best practices. 

Can you confirm that that definition will not declare the variable at all? Thanks

**Comment by rpelisse** [comment]
@rmarting Can you review this PR (and maybe test it a bit)? 

Thanks!

File: roles/amq_streams_broker/tasks/service.yml (hunk 1 above)
Comment (rmarting) [reply roles/amq_streams_broker/tasks/service.yml:29]: @rpelisse Added this comment :point_up:

**Review by rmarting — CHANGES_REQUESTED** [review]

File: roles/amq_streams_broker/tasks/service.yml (hunk 1 above)
Comment (rpelisse) [reply roles/amq_streams_broker/tasks/service.yml:29]: That'll be great indeed :) CI tells us it's not breaking anything, but we do need to confirm it does what we expect it to do!

**Pull request merged**